docs/designers-developers/developers/tutorials/javascript/loading-javascript.md

```diff
@@ -0,0 +1,48 @@
+
+## Loading JavaScript
+
+With the plugin in place, you can add the code that loads the JavaScript. This methodology follows the standard WordPress procedure of enqueuing scripts, see [enqueuing section of the Plugin Handbook](https://developer.wordpress.org/plugins/javascript/enqueuing/).
+
+Add the following code to your `myguten-plugin.php` file:
+
+```php
+function myguten_enqueue() {
+	wp_enqueue_script( 'myguten-script',
+		plugins_url( 'myguten.js', __FILE__ ),
+		array( 'wp-blocks', 'wp-element' )
```

Comment:
I dig this step-by-step approach! It's very clear.

For the sake of creating a _Minimum Viable Plugin_, how about removing these dependencies. They're not required just yet to write to the console.